pages/common/ngrep.md

```diff
@@ -0,0 +1,23 @@
+# ngrep
+
+> Filter network traffic packets using regular expressions.
+
+- Capture traffic of all interfaces:
+
+`ngrep -d any`
```

Comment:
I must misunderstood the token syntax, I will wrap "eth0", "22" with {{ }}, but "any" is a keywork in ngrep, thx.